axelor-base/src/main/java/com/axelor/apps/base/web/ProductPackagingController.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2005-2025 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.axelor.apps.base.web;
+
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.base.db.ProductPackaging;
+import com.axelor.apps.base.exceptions.BaseExceptionMessage;
+import com.axelor.apps.base.service.ProductPackagingService;
+import com.axelor.apps.base.service.exception.TraceBackService;
+import com.axelor.i18n.I18n;
+import com.axelor.inject.Beans;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+
+public class ProductPackagingController {
+
+  public void computePackagingMaxTotalWeight(ActionRequest request, ActionResponse response) {
+    try {
+      ProductPackaging packaging = request.getContext().asType(ProductPackaging.class);
+      response.setValue(
+          "packagingMaxTotalWeight",
+          Beans.get(ProductPackagingService.class).computePackagingMaxTotalWeight(packaging));
+    } catch (AxelorException e) {
+      TraceBackService.trace(e);
+    }
+  }
+
+  public void validateTotalWeight(ActionRequest request, ActionResponse response) {
+    try {
+      ProductPackaging packaging = request.getContext().asType(ProductPackaging.class);
+      if (!Beans.get(ProductPackagingService.class).validateTotalWeight(packaging)) {
+        response.setError(I18n.get(BaseExceptionMessage.PACKAGING_MAX_WEIGHT_EXCEEDED));
+      }
```

Comment:
You can keep this without the try catch,  the base interceptor will automatically catch the exception.